test/cli/inline-suppress_test.py

```diff
@@ -456,4 +456,44 @@ def test_unmatched_cfg():
         '{}cfg.c:14:0: information: Unmatched suppression: id [unmatchedSuppression]'.format(__proj_inline_suppres_path),
     ]
     assert stdout == ''
-    assert ret == 0, stdout
\ No newline at end of file
+    assert ret == 0, stdout
+
+def test_xml_report_stdout(tmpdir):
+    args = [
+        '-q',
+        '--inline-suppr',
```

Comment:
I don't remember if some `-j2` and `-j1` is injected somehow in the CI. but it would make sense to test that this works with multiple jobs also.